src/test/java/com/atlassian/bitbucket/jenkins/internal/status/BuildStatusPosterTest.java

```diff
@@ -71,7 +72,12 @@ public void setup() {
         when(buildStatusFactory.createRichBuildStatus(run)).thenReturn(buildStatus);
         when(buildStatusFactory.createLegacyBuildStatus(run)).thenReturn(buildStatus);
     }
-
+    
+    @After
+    public void cleanup() {
+        System.setProperty("bitbucket.status.disable", "");
```

Comment:
I hope these tests are run sequentially rather than in parallel.

Reply:
Good point thanks!